ajax/libs/sir-trevor-js/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "sir-trevor-js",
+  "filename": "sir-trevor.min.js",
+  "version": "0.6.0-beta9",
+  "description": "Rich content editing entirely re-imagined for the web",
+  "homepage": "https://madebymany.github.io/sir-trevor-js/",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:madebymany/sir-trevor-js.git"
+  },
+  "keywords": [
+    "content editing",
+    "rich content editor",
+    "Sir Trevor",
+    "SirTrevor"
```

Comment:
@PeterDaveHello people aren't really sure how to reference the editor so we have both the one word and joined term in there.